pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1033,9 +1052,9 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 		err                    error
 	)
 	if c.clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
-		// Check internal FSS map.
+		// first check hard coded FSS map. these are GA'ed features
 		c.internalFSS.featureStatesLock.RLock()
-		if flag, ok := c.internalFSS.featureStates[featureName]; ok {
+		if flag, isReleased := c.releasedVanillaFSS[featureName]; isReleased {
```

Comment:
if `isReleased` is true then we should return true instead of reading fss from internalFSS

Reply:
Got it. This would have parsed the feature name in the hardcoded map which would still have returned true but yes that extra parsing is not needed here.